engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/runtime/Context.java

```diff
@@ -0,0 +1,27 @@
+package org.enso.interpreter.node.expression.builtin.runtime;
+
+import org.enso.interpreter.dsl.BuiltinType;
+import org.enso.interpreter.node.expression.builtin.Builtin;
+import org.enso.interpreter.runtime.callable.atom.AtomConstructor;
+
+import java.util.List;
+
+@BuiltinType
+public class Context extends Builtin {
+  @Override
+  protected List<Cons> getDeclaredConstructors() {
+    return List.of(new Cons(INPUT_CONTEXT), new Cons(OUTPUT_CONTEXT));
+  }
+
+  public static String INPUT_CONTEXT = "Input";
```

Comment:
Missing `final`. Name of the field reads as `Context.INPUT_CONTEXT` from an external class usage - isn't the "context" duplicated?